__tests__/swagger.yml

```diff
@@ -1,59 +1,77 @@
-swagger: "2.0"
+openapi: 3.0.0
 
 info:
   title: Parrot Test Api
   description: test api
-  version: 1.0.0
+  contact: {}
+  version: '1.0.0'
 
-basePath: /parrot-test
-schemes:
-  - http
-
-produces:
-  - application/json
-  - text/plain
-consumes:
-  - application/json
+servers:
+- url: http://{defaultHost}
+  variables:
+    defaultHost:
+      default: www.example.com/parrot-test # todo figure out basePath
```

Comment:
this is the last remaining thing i need to figure out (ie, how do i set the `/parrot-test` base path so that the middleware parses it correctly)